### ISSUE TITLE
Which translation file takes priority?

### ISSUE THREAD

**Issue by savolai** [aschet/linguistmerge#1]
Hi there,
I finally got around to trying this out. The UI seems really clean, appreciate it.
However, I ran into not being sure how to get it to prioritize translations in one file over another. 

> If multiple of your source files contain conflicting, nonempty, translations for the same translation units (i.e classname+source text content), make sure to load the file that has the translations you want, last.

In Merge-Qt-TS-files I made it so that the file added last would get priority when two files contain translations for the same string. In your app this approach didn't seem to work and I can't find any mention of how that's supposed to work. Didn't study your source for now since I knew how to do this in my app, but thought you might appreciate the heads up.

**Comment by aschet** [comment]
> However, I ran into not being sure how to get it to prioritize translations in one file over another.

I agree, the inner workings of the merge strategy are not well expressed via the UI at the moment. The priority is determined by the list order and the message status. This means that, evaluated from top to bottom, the first accepted message ends up in the merged file.
